codebase/superdarn/src.dlm/rposdlm.1.9/rposdlm.c

```diff
@@ -511,20 +528,23 @@ static IDL_VPTR IDLRadarYMDHMSGetSite(int argc,IDL_VPTR *argv) {
 
   isite=IDLRadarMakeSite(&vsite);
 
+  isite->tval=status->site[s].status;
```

Comment:
This line is causing a compilation error (see below). It compiles ok with the following change: 

```suggestion
  isite->tval=iradar->site[s].status;
```

```
================================================================================
Compiling Binary:/rst/codebase/superdarn/src.dlm/rposdlm.1.9
================================================================================
make clean
rm -f *.o
make
make.version /home/emma/gitrepos/rst/codebase/superdarn/src.dlm/rposdlm.1.9/..
cc -fPIC -Wall -pedantic -O3 -D_GNU_SOURCE -D_LINUX -I/rst/include/base -I/rst/include/general -I/rst/include/superdarn -I/usr/local/itt/idl/external/include -c rposdlm.c    
rposdlm.c: In function ‘IDLRadarYMDHMSGetSite’:
rposdlm.c:531:15: error: ‘status’ undeclared (first use in this function); did you mean ‘statx’?
  531 |   isite->tval=status->site[s].status;
      |               ^~~~~~
      |               statx
rposdlm.c:531:15: note: each undeclared identifier is reported only once for each function it appears in
make: *** [/rst/build/make/makedlm.linux:19: rposdlm.o] Error 1
Compilation Aborted.
```

Reply:
@ecbland thanks for testing!